lib/tasks/feeds/event.rake

```diff
@@ -3,34 +3,14 @@ namespace :feeds do
   task event: :environment do
     if ENV['REPORT_ON_DATE']
       report_on_date = Date.parse(ENV['REPORT_ON_DATE'])
-      updated_at_from = report_on_date.beginning_of_day
-      updated_at_to = report_on_date.end_of_day
+      created_at_from = report_on_date.beginning_of_day
+      created_at_to = report_on_date.end_of_day
 
-      feed = Feeds::Event.new(updated_at_from, updated_at_to).call
+      feed = Feeds::Event.new(created_at_from, created_at_to).call
       CloudDataFeed.new.write(feed, 'events.jsonl', report_on_date)
     else
       feed = Feeds::Event.new.call
       CloudDataFeed.new.write(feed, 'events.jsonl')
     end
   end
-
-  desc 'Exports the JSON starting from a certain date (inclusive)'
```

Comment:
just wondering why this task is no longer required?

Reply:
I could have left it but I actually don't think it will be used now that we can regenerate reports for a specific day. This was originally needed to backfill all events after adding a missing supplier id to all events.

Reply:
👍